charts/home-assistant/README.md

```diff
@@ -85,7 +85,8 @@ The following tables lists the configurable parameters of the Home Assistant cha
 | `git.syncPath`                                  | Git sync path                                                                                                                                                                                                                             | `/config`                           |
 | `git.keyPath`                                   | Git ssh key path                                                                                                                                                                                                                          | `/root/.ssh`                        |
 | `git.user.name`                                 | Human-readable name in the “committer” and “author” fields                                                                                                                                                                                | ``                                  |
-| `git.user.email`                                | Email address for the “committer” and “author” fields                                                                                                                                                                                     | ``                                  |
+| `git.user.email`                                | Email address for the “committer” and “author” fields                    | ``                                 |                                                      
+| `git.configSubPath`                             | An optional subPath for the git configuration volumeMount                | ``                                |
```

Comment:
Should we also add this to the `values.yaml` file so folks know that it's an option that can be set?  Generally if something is a configurable value, it will have a stub (or commented stub) in `values.yaml`

Reply:
I'm not completely sure if `git` is the best key to place this under, as this setting would also be applicable when not using the `git` settings at all. We (probably?) also don't set the config `mountPath` to `git.syncPath` for similar reasons.

I understand that it needs to go somewhere, but maybe it is better suited under the `persistence` key or an entirely new key?

Reply:
this one is specifically for the git-sync config volume mount.